lerna.json

```diff
@@ -1,15 +1,6 @@
 {
   "npmClient": "npm",
-  "packages": [
```

Comment:
I was not aware that we can remove `packages` in `lerna.json` when using workspaces. Would you mind removing it in https://github.com/open-telemetry/opentelemetry-js/ as well?